<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestingSettings</class>
 <widget class="QDialog" name="TestingSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Testing settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../_resources/editor.qrc">
    <normaloff>:/appicon/app.ico</normaloff>:/appicon/app.ico</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <widget class="QGroupBox" name="extraSettings">
     <property name="title">
      <string>Extra settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QGroupBox" name="cheats">
        <property name="title">
         <string>Cheats</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QCheckBox" name="ex_god">
           <property name="toolTip">
            <string>Playable character will not take damage on dangerous contacts and will not burn in the lava.</string>
           </property>
           <property name="text">
            <string>God mode</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="ex_flyup">
           <property name="toolTip">
            <string>Allows you to fly up to the space!</string>
           </property>
           <property name="text">
            <string>Unlimited flying</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="ex_chuck">
           <property name="toolTip">
            <string>Allows you to destroy any objects with no exceptions.</string>
           </property>
           <property name="text">
            <string>Bulldozer mode</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="ex_freedom">
           <property name="toolTip">
            <string>Allows you to walk everywhere on the world map without limiting by paths.</string>
           </property>
           <property name="text">
            <string>World freedom</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_7">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QGroupBox" name="initialState">
        <property name="title">
         <string>Initial state</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Start with stars:</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QSpinBox" name="ex_starsNum">
              <property name="maximum">
               <number>1000</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_6">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="9" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QGroupBox" name="debug">
        <property name="title">
         <string>Debug</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QCheckBox" name="ex_debug">
           <property name="text">
            <string comment="Enable printing of the debug information.">Debug info</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="ex_showFPS">
           <property name="text">
            <string>Show frame speed</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="ex_physdebug">
           <property name="text">
            <string comment="Enable debug drawing of physical objects (draw all hit boxes).">Physics debug</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_8">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="numOfPlayers">
     <property name="title">
      <string>Number of players</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QRadioButton" name="np_1p">
        <property name="text">
         <string>1 player</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="np_2p">
        <property name="text">
         <string>2 player</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>101</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="playerSettings">
     <property name="title">
      <string>Player settings</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QGroupBox" name="setupP1">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>144</height>
         </size>
        </property>
        <property name="title">
         <string>1 Player settings</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string comment="Character">PLSET_Character</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QComboBox" name="p1_character">
           <item>
            <property name="text">
             <string notr="true">ch1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">ch2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">ch3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">ch4</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">ch5</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string comment="Character">PLSET_State</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QComboBox" name="p1_state">
           <item>
            <property name="text">
             <string notr="true">st1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">st2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">st3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">st4</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">st5</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">st6</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">st7</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="8" column="0">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="5" column="0">
          <widget class="QComboBox" name="p1_vehicleID">
           <item>
            <property name="text">
             <string notr="true">[No vehicle]</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">Vehicle 1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">Vehicle 2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">Vehicle 3</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QComboBox" name="p1_vehicleType">
           <item>
            <property name="text">
             <string notr="true">[Unknown type]</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">Type 1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">Type 2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">Type 3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">Type 4</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">Type 5</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">Type 6</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">Type 7</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">Type 8</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string comment="translate as &quot;Vehicle&quot;">PLSET_VehicleID</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string comment="translate as &quot;Vehicle type&quot;">PLSET_VehicleType</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="setupP2">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>0</height>
         </size>
        </property>
        <property name="title">
         <string>2 Player settings</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="8" column="0">
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="0">
          <widget class="QComboBox" name="p2_state">
           <item>
            <property name="text">
             <string notr="true">st1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">st2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">st3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">st4</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">st5</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">st6</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">st7</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string comment="Character">PLSET_State</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string comment="Character">PLSET_Character</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QComboBox" name="p2_character">
           <item>
            <property name="text">
             <string notr="true">ch1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">ch2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">ch3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">ch4</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">ch5</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QComboBox" name="p2_vehicleID">
           <item>
            <property name="text">
             <string notr="true">[No vehicle]</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">Vehicle 1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">Vehicle 2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">Vehicle 3</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string comment="translate as &quot;Vehicle&quot;">PLSET_VehicleID</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string comment="translate as &quot;Vehicle type&quot;">PLSET_VehicleType</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QComboBox" name="p2_vehicleType">
           <item>
            <property name="text">
             <string notr="true">[Unknown type]</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">Type 1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">Type 2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">Type 3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">Type 4</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">Type 5</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">Type 6</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">Type 7</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">Type 8</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <spacer name="verticalSpacer_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../_resources/editor.qrc"/>
  <include location="../../_resources/editor.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TestingSettings</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>371</y>
    </hint>
    <hint type="destinationlabel">
     <x>153</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
